examples/tests/client-tests.js

```diff
@@ -0,0 +1,173 @@
+'use strict';
+
+fetch = () => {
```

Comment:
please include a comment on what we're doing here and why - hiding the original fetch